proto/encore/parser/meta/v1/meta.proto

```diff
@@ -287,7 +288,31 @@ message CacheCluster {
   string            eviction_policy = 4; // redis eviction policy
 
   message Keyspace {
-    // TODO(andre) add more fields
-    schema.v1.Type key_type     = 1; // The type of the key
+    schema.v1.Type key_type     = 1;
+    schema.v1.Type value_type   = 2;
+    string         service      = 3;
+    string         doc          = 4;
+    Path           path_pattern = 5;
   }
 }
+
+message Metric {
+  string            name         = 1; // the name of the metric
+  schema.v1.Builtin value_type   = 2;
+  string            doc          = 3; // the doc string
+  MetricKind        kind         = 4;
```

Comment:
@eandre I renamed this field to `kind` to differentiate it from the metric value type.